services/movie_session.py

```diff
@@ -0,0 +1,65 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: str

```

Comment:
Consider using consistent data types for IDs. `cinema_hall_id` is a string here, but it should be an integer for consistency with `movie_id`.